department-of-veterans-affairs/abd-vro
# Support Team CC Docker Image Publishing

## What was the problem?

Prior to this PR, automated scripting for defining variables for docker images did not support docker files in subdirectories.

Associated tickets or Slack threads:


## How does this fix it?[^1]

Alters the `image-name.sh` script to strip out slashes in the bash variable names and to replace slashes with dashes in the docker image names

## How to test this PR
- New script has been tested by running it and getting the new output file

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
